examples/web-search-crawl.py

```diff
@@ -8,56 +8,39 @@
 import os
 from typing import Union
 
-from rich import print
+from ollama import Client, WebFetchResponse, WebSearchResponse
 
-from ollama import Client, WebCrawlResponse, WebSearchResponse
 
-
-def format_tool_results(results: Union[WebSearchResponse, WebCrawlResponse]):
+def format_tool_results(results: Union[WebSearchResponse, WebFetchResponse]):
   if isinstance(results, WebSearchResponse):
-    if not results.success:
-      error_msg = ', '.join(results.errors) if results.errors else 'Unknown error'
-      return f'Web search failed: {error_msg}'
-
     output = []
-    for query, search_results in results.results.items():
-      output.append(f'Search results for "{query}":')
-      for i, result in enumerate(search_results, 1):
-        output.append(f'{i}. {result.title}')
-        output.append(f'   URL: {result.url}')
-        output.append(f'   Content: {result.content}')
-        output.append('')
```

Comment:
I don't think this block should've changed. Check string output before and after